tests/test_app.py

```diff
@@ -132,58 +132,105 @@ def test_manifest_get_not_found(client):
         assert json_data["detail"] == "Content for ubi_repo_id not found"
 
 
-def test_manifest_post(client):
+@mock.patch("celery.app.task.Task.apply_async")
+@mock.patch("ubi_manifest.app.api.get_items_for_depsolving")
```

Comment:
As I stated in the comment for test_app_utils.py ... this is exactly something you don't want to patch.
What happens if you change the return value of `get_items_for_depsolving()` - this test will still pass and you won't be forced to update this test and bugs can stay hidden.